pkg/api/operators/apis/trial/v1alpha2/trial_types.go

```diff
@@ -29,6 +29,9 @@ type TrialSpec struct {
 	// and let the corresponding resource controller (e.g. tf-operator) handle
 	// the rest.
 	RunSpec string `json:"runSpec,omitempty"`
+
+	// Raw text for the metrics collector spec. This must be a CronJob object.
+	MetricsCollectorSpec string `json:"metricsCollectorSpec,omitempty"`
```

Comment:
add Objective name and type  to spec to figure out the best values?

Reply:
Added ObjectiveSpec to Trial.